lib/utils/Logger.js

```diff
@@ -20,10 +20,20 @@ class Logger {
          * error : error only
          * verbose : command + data + result
          */
+
+
+        /**
+         * logLevel verification
+         */
+        if (options.logLevel != 'silent' && options.logLevel != 'command' && options.logLevel != 'result' && options.logLevel != 'error' && options.logLevel != 'verbose') {
```

Comment:
I think it is easier to have all log options in an array and check if it includes the given logLevel